gulpfile.js

```diff
@@ -51,10 +51,24 @@ gulp.task('webpack', ['clean'], function() {
     ;
 });
 
+// What all processing needs to be done ?
```

Comment:
do we need this comment?
the text of the comment is confusing

Reply:
no we don't need this 
changed it for its funtionality